geode-core/src/test/java/org/apache/geode/test/dunit/IgnoredException.java

```diff
@@ -110,7 +110,7 @@ public void run() {
     if (this.vm != null) {
       vm.invoke(removeRunnable);
     } else {
-      Invoke.invokeInEveryVM(removeRunnable);
+      Invoke.invokeInEveryVM(null, removeRunnable, true);
```

Comment:
I like that you have added names to invocations in other places.  Is there an appropriate name for this invocation?  Maybe "remove ignored exception"?